cosmos/dbt/graph.py

```diff
@@ -76,6 +76,10 @@ def name(self) -> str:
         """
         return self.resource_name.replace(".", "_")
 
+    @property
+    def owner(self) -> str:
+        return str(self.config.get("meta", {}).get("owner", "airflow"))
```

Comment:
An alternative to hard-code `airflow` could be to retrieve the default value from Airflow:
```
from airflow.configuration import conf

default_owner = conf.get("operators", "default_owner")
```

If you want to follow this path, please define `default_owner` in `settings.py`, together with other configurations.

Reply:
I'll check it out tonight

Reply:
if there is no owner in dbt config 
It would be better to specify the owner for the Airflow DAG.

Reply:
Thanks a lot, @wornjs !